ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -195,6 +196,17 @@ public void SetPlaceholder(PasswordBox view, string placeholder)
         }
 
         /// <summary>
```

Comment:
Adding the placeholderTextColor setter to the ReactNative view managers because it is assumed that since we added the property to the javascript that we'd likely need the symmetric change in the C#... Since placeholderTextColor is not supported with the UWP built in PlaceholderText property we throw a not implemented exception...